spec/models/billing_spec.rb

```diff
@@ -0,0 +1,27 @@
+RSpec.describe Billing, type: :model do
```

Comment:
```suggestion
RSpec.describe Address, type: :model do
```
No need to test Billing and Shipping separately, unless they have some different behavior.